lbrynet/conf.py

```diff
@@ -1,116 +1,99 @@
-    default_lbryum_dir = os.path.join(os.path.expanduser("~"), ".lbryum")
-elif sys.platform.startswith("win"):
+    default_download_directory = os.path.join(os.path.expanduser('~'), 'Downloads')
+    default_data_dir = user_data_dir('LBRY')
+    default_lbryum_dir = os.path.join(os.path.expanduser('~'), '.lbryum')
+elif sys.platform.startswith('win'):
     platform = WINDOWS
     from lbrynet.winhelpers.knownpaths import get_path, FOLDERID, UserHandle
+
     default_download_directory = get_path(FOLDERID.Downloads, UserHandle.current)
     default_data_dir = os.path.join(
-        get_path(FOLDERID.RoamingAppData, UserHandle.current), "lbrynet")
+        get_path(FOLDERID.RoamingAppData, UserHandle.current), 'lbrynet')
     default_lbryum_dir = os.path.join(
-        get_path(FOLDERID.RoamingAppData, UserHandle.current), "lbryum")
+        get_path(FOLDERID.RoamingAppData, UserHandle.current), 'lbryum')
 else:
     platform = LINUX
-    default_download_directory = os.path.join(os.path.expanduser("~"), 'Downloads')
-    default_data_dir = os.path.join(os.path.expanduser("~"), ".lbrynet")
-    default_lbryum_dir = os.path.join(os.path.expanduser("~"), ".lbryum")
-
+    default_download_directory = os.path.join(os.path.expanduser('~'), 'Downloads')
+    default_data_dir = os.path.join(os.path.expanduser('~'), '.lbrynet')
+    default_lbryum_dir = os.path.join(os.path.expanduser('~'), '.lbryum')
 
-ICON_PATH = "icons" if platform is WINDOWS else "app.icns"
+ICON_PATH = 'icons' if platform is WINDOWS else 'app.icns'
 
 
-class Settings(object):
-    """A collection of configuration settings"""
-    __fixed = []
-    _excluded = ['get_dict', 'update']
-
-    def __iter__(self):
-        for k in self.__dict__.iterkeys():
-            if k.startswith('_') or k in self._excluded:
-                continue
-            yield k
-
-    def __getitem__(self, item):
-        assert item in self, IndexError
-        return self.__dict__[item]
-
-    def __setitem__(self, key, value):
-        assert key in self and key not in self.__fixed, KeyError(key)
-        self.__dict__[key] = value
-
-    def __contains__(self, item):
-        return item in iter(self)
-
-    def get_dict(self):
-        return {k: self[k] for k in self}
-
-    def update(self, updated_settings):
-        for k, v in updated_settings.iteritems():
-            try:
-                self.__setitem__(k, v)
-            except (KeyError, AssertionError):
-                pass
+def server_port(server_and_port):
+    server, port = server_and_port.split(':')
+    return server, int(port)
 
 
 class Env(envparse.Env):
     """An Env parser that automatically namespaces the variables with LBRY"""
-    NAMESPACE = 'LBRY_'
+
     def __init__(self, **schema):
         self.original_schema = schema
         my_schema = {
             self._convert_key(key): self._convert_value(value)
             for key, value in schema.items()
-        }
+            }
```

Comment:
i've been fighting with pycharm to set up autoformat correctly, but its still not great. if you have a config that works, send it over. I'd love to use it